src/self_driving/planning/pilot_util.cpp

```diff
@@ -166,7 +132,8 @@ double PilotUtil::ComputeCost(common::ManagedPointer<Pilot> pilot, common::Manag
       prev_qid = pipeline_to_pred.first.first;
     }
   }
-  double total_cost = 0, num_queries = 0;
+  double total_cost = 0;
+  double num_queries = 0;
```

Comment:
Can you have a fractional number of queries?

Reply:
Yes. It's possible from the forecasting.